### ISSUE TITLE
All icons disappear!

### ISSUE THREAD

**Issue by flyisland** [pors/reactionic-kitchensink#16]
I have just downloaded and ran this example by following the README, and it seems that all icons disappear!

![2016-06-18 12_11_32-window](https://cloud.githubusercontent.com/assets/547085/16169007/c8169920-354f-11e6-8453-4ee5578cd5df.png)

**Comment by gabrielseco** [comment]
I have downloaded the repo and I couldn't reproduce this, it works for me.

The screenshot you show is in an android device right?

**Comment by flyisland** [comment]
@gabrielseco 
I'm using Windows 10, and used Chrome to access "localhost:8080", and the screenshot is captured by chrome with "device mode" in "developer tool"

And I have just try it again with centos 6.5 and firefox, got the same result.

**Comment by flyisland** [comment]
btw, I'm using webpack by following:

```
cd reactionic-kitchensink
npm install
webpack
webpack-dev-server --content-base build
```

**Comment by gabrielseco** [comment]
I was using meteor, I tried using webpack and it's loading the icons.

I'll try from a windows device.

<img width="808" alt="captura de pantalla 2016-06-18 a las 18 29 07" src="https://cloud.githubusercontent.com/assets/7215480/16172161/b02a5eae-3582-11e6-814d-fb460f8d3d63.png">

**Comment by flyisland** [comment]
yes, it's quite confused, since all the functions is OK, just lost these icons.
![localhost-8080- galaxy s5](https://cloud.githubusercontent.com/assets/547085/16174516/d5172d94-35f7-11e6-89fe-4c9416952b6a.png)

Is the following icon's style content is right?
![2016-06-19 08_31_14-reactionic react ionic](https://cloud.githubusercontent.com/assets/547085/16174529/3f20ff3a-35f8-11e6-9701-4f98da485ba2.png)
![2016-06-19 08_26_05-new notification](https://cloud.githubusercontent.com/assets/547085/16174517/da1d7b9a-35f7-11e6-9a92-2adfabaac416.png)

**Comment by gabrielseco** [comment]
If you open the chrome with developer tools, and you click in the tab network.

It should load ionicons fonts.

<img width="732" alt="captura de pantalla 2016-06-19 a las 10 27 53" src="https://cloud.githubusercontent.com/assets/7215480/16176207/aba58210-3608-11e6-8009-89c5fb12c42c.png">

**Comment by flyisland** [comment]
It seems that all fonts are missing! And idea to fix it?
![2016-06-19 23_24_55-reactionic react ionic](https://cloud.githubusercontent.com/assets/547085/16178243/3ee1a920-3675-11e6-822c-163bd729bacb.png)

**Comment by gabrielseco** [comment]
In the next route app/public/fonts there should be four files with the fonts.

If there are present, you could try removing reactionic from node-modules and installing it again

**Comment by flyisland** [comment]
@gabrielseco I re-downloaded the `https://github.com/reactionic/reactionic-kitchensink/archive/master.zip`, and followed your instruction to build the project, but got the same result again.

Since the Chrome says `/fonts/ionicons.ttf` is not found, so I renamed the file `\build\fonts`, and copy the `fonts` folder from `node_modules\reactionic\dist\scss\` into `\build`, just like below:

```
D:\DevProjects\reactionic-demo\build>dir
 Volume in drive D is Data
 Volume Serial Number is 4AE4-3C93

 Directory of D:\DevProjects\reactionic-demo\build

2016-06-20  11:28    <DIR>          .
2016-06-20  11:28    <DIR>          ..
2016-06-20  10:42    <DIR>          fonts
2016-05-05  07:12                42 fonts.old
2016-05-05  07:12               421 index.html
2016-06-19  23:17         2,004,294 index.js
               3 File(s)      2,004,757 bytes
               3 Dir(s)  75,198,185,472 bytes free

D:\DevProjects\reactionic-demo\build>dir fonts
 Volume in drive D is Data
 Volume Serial Number is 4AE4-3C93

 Directory of D:\DevProjects\reactionic-demo\build\fonts

2016-06-20  10:42    <DIR>          .
2016-06-20  10:42    <DIR>          ..
2016-06-19  23:00           120,724 ionicons.eot
2016-06-19  23:01           333,834 ionicons.svg
2016-06-19  23:01           188,508 ionicons.ttf
2016-06-19  23:01            67,904 ionicons.woff
               4 File(s)        710,970 bytes
               2 Dir(s)  75,198,185,472 bytes free

D:\DevProjects\reactionic-demo\build>
```

And now the icons appears!

So this is a workaround, but we still need an official solution to fix it.

**Comment by gabrielseco** [comment]
I tried today in a windows and I could reproduce the problem.

It seems that the  Symbolic Link in the build called fonts it's not working in windows.

@pors how about we put directly the fonts in the build to save the windows problem or it occurs you another idea to solve this.

**Comment by pors** [comment]
@gabrielseco good find! Hmm, yes. Windows does support symlinks I think? It would be cleaner if we could have symlinks that work everywhere, but I have never looked into that. You guys have any ideas?

If we can't make a symlink work on Windows then copying it into the build directory is OK I guess. We have to remember to update these files when we update reactionic itself.

**Comment by gabrielseco** [comment]
I'll give it a try this week to this issue and #17.

People had some issues accessing from windows, I'll try from there to see what I can do.

**Comment by pors** [comment]
Awesome

**Comment by flyisland** [comment]
I think the `build` folder should work without depending on other assets, so i could make the build on a linux box  and copy the `build` folder to a windows machine, it should still be okay

**Comment by aminur** [comment]
I am still facing this issue even after renaming 'fonts' in 'build' directory to 'fonts.old' and pasting in a copy of the 'fonts' directory from node_modules\reactionic\dist\scss\

on Windows 7
